<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pswdmgr-gui</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>490</width>
     <height>260</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTableWidget" name="PswdsTable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Pswd_File"/>
    <addaction name="actionWriteout"/>
    <addaction name="actionWriteoutToFile"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_New_Pswd_File"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionAdd_Password"/>
    <addaction name="actionRemove_Site_Pswd"/>
    <addaction name="separator"/>
    <addaction name="actionChange_Password"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuManage"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Pswd_File">
   <property name="text">
    <string>Open Pswd File...</string>
   </property>
  </action>
  <action name="actionAdd_Password">
   <property name="text">
    <string>Add Site Pswd</string>
   </property>
  </action>
  <action name="actionRemove_Site_Pswd">
   <property name="text">
    <string>Remove Site Pswd</string>
   </property>
  </action>
  <action name="actionWriteoutToFile">
   <property name="text">
    <string>Write To File...</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="text">
    <string>Change Password</string>
   </property>
  </action>
  <action name="actionWriteout">
   <property name="text">
    <string>Writeout</string>
   </property>
  </action>
  <action name="actionCreate_New_Pswd_File">
   <property name="text">
    <string>Create New Pswd File...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelp</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_Password</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>AddSitePswd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_Site_Pswd</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>RemoveSitePswd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Pswd_File</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenPswdFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWriteoutToFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>WriteoutToFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChange_Password</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ChangePswd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PswdsTable</sender>
   <signal>cellChanged(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>CellChanged(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWriteout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>WriteoutPswds()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PswdsTable</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>CellEntered(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCreate_New_Pswd_File</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>CreateNewPswdFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenAbout()</slot>
  <slot>OpenHelp()</slot>
  <slot>OpenPswdFile()</slot>
  <slot>WriteoutPswds()</slot>
  <slot>AddSitePswd()</slot>
  <slot>RemoveSitePswd()</slot>
  <slot>ChangePswd()</slot>
  <slot>CellChanged(int,int)</slot>
  <slot>WriteoutToFile()</slot>
  <slot>CellEntered(int,int)</slot>
  <slot>CreateNewPswdFile()</slot>
 </slots>
</ui>
